include/contractor/contract_excludable_graph.hpp

```diff
@@ -94,7 +92,6 @@ inline auto contractExcludableGraph(ContractorGraph contractor_graph_,
     return GraphAndFilter{QueryGraph{num_nodes, edge_container.edges},
                           edge_container.MakeEdgeFilters()};
 }
-} // namespace contractor
 } // namespace osrm
```

Comment:
nit. would be great to reflect this in such comments too(if possible to do in automated fashion ofc)

Reply:
fair point. I am afraid those are hard to do in an automated way, tho.

Reply:
Then I think it is not big problem if we keep it like this

Reply:
I spoke too early. The latest push to this PR should fix the closing namespace annotation.